packages/flutter_tools/test/general.shard/ios/devices_test.dart

```diff
@@ -97,6 +97,7 @@ void main() {
         'device-123',
         artifacts: mockArtifacts,
         fileSystem: mockFileSystem,
+        logger: mockLogger,
```

Comment:
If you're not specifically mocking some functionality of the logger for testing, use BufferLogger.test

Reply:
Yup, I'm resolving the merge conflict now and cleaning this up.

Reply:
resolved merge and removed all `mockLogger` instances, PTAL.